input/templates/partials/catalog/reserve/initialize-map-scripts.hbs

```diff
@@ -0,0 +1,79 @@
+<script>
+  $(window).load(function() {
```

Comment:
I think google made JS inline just for demo purposes. As you see they also do CSS inline in their examples. I did not find exact statement where they discourage loading JS code as resource. Also i do not see in this file usage of handlebars.
It is not big issue but i think it makes things less intuitive if scripts can be handlebar templates also and it makes generated HTML hard to read.